server/monitor.go

```diff
@@ -247,6 +266,14 @@ func (s *Server) Connz(opts *ConnzOptions) (*Connz, error) {
 		// Gather all open clients.
 		if state == ConnOpen || state == ConnAll {
 			for _, client := range s.clients {
+				// If we have an account specified we need to filter.
+				if acc != "" && client.acc.Name != acc {
```

Comment:
I think that a client stored in `s.clients` should always have `acc` set (even if with global account to start with). So I guess it is ok not to check for `client.acc != nil`...

Reply:
Yes that is correct, but could add it.

Reply:
Personal choice: when I know it can't be nil, I don't check. I more often have a comment when it can be nil and that's why I check, etc..

Reply:
I added it in, please take a look when you have some time.

Reply:
I did review. LGTM.